acquia/acquia_cms
# ACMS-964: Social media items on the footer of the page are identical

**Motivation**

Fixes #ACMS-964(https://backlog.acquia.com/browse/ACMS-964)

**Proposed changes**

Fix social link icon

**Alternatives considered**

NA

**Testing steps**


- Install site, visit home page
- check social share link in footer section and make sure all links and are correct.

**Merge requirements**
- [_Bug_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
